service/src/main/java/bio/terra/cbas/dependencies/ecm/EcmClient.java

```diff
@@ -0,0 +1,45 @@
+package bio.terra.cbas.dependencies.ecm;
+
+import bio.terra.cbas.config.EcmServerConfiguration;
+import bio.terra.externalcreds.api.OauthApi;
+import bio.terra.externalcreds.api.PublicApi;
+import bio.terra.externalcreds.client.ApiClient;
+import org.springframework.stereotype.Component;
+
+@Component
+public class EcmClient {
+  private final EcmServerConfiguration ecmServerConfiguration;
+
+  public EcmClient(EcmServerConfiguration ecmServerConfiguration) {
+    this.ecmServerConfiguration = ecmServerConfiguration;
+  }
+
+  public ApiClient apiClient() {
```

Comment:
Would we ever want an unauthorized ecm?

Reply:
Yeah, we're currently using it for the scheduled health check

Reply:
Of course!